hector_gazebo_plugins/src/gazebo_ros_force_based_move.cpp

```diff
@@ -375,17 +375,17 @@ namespace gazebo
       const double distY = linear_vel_y * timeSeconds;
       const double distChange = std::sqrt(distX * distX + distY * distY);
       const double angleDist = std::acos(distX / distChange);
-      const double angleChange = angular_vel * timeSeconds;
+      const double angleDriveDirection = angular_vel * timeSeconds;
```

Comment:
Thank you for the quick update but I meant the `angleDist` variable which is not a distance but the direction the vehicle is driving due to its x and y velocities.
`angleChange` is a valid name as it is the difference of the angle since the last timestep.

Reply:
My bad, sorry! I went too quickly and renamed the wrong variable. Fixed by https://github.com/tu-darmstadt-ros-pkg/hector_gazebo/pull/71/commits/eff04c661c12aa3275fe65950c5deab3b4d594bd.